src/parser/jobs/vpr/modules/RattlingCoil.tsx

```diff
@@ -97,19 +116,31 @@ export class RattlingCoil extends CoreGauge {
 		}))
 
 		this.checklist.add(new Rule({
-			name: <Trans id="vpr.rattlingcoil.usage.title"> <DataLink action="UNCOILED_FURY"/> Usage</Trans>,
+			name: <Trans id="vpr.rattlingcoil.usage.title"> <DataLink action="UNCOILED_FURY"/>,  <DataLink action="UNCOILED_TWINBLOOD"/> &  <DataLink action="UNCOILED_TWINFANG"/> Usage</Trans>,
```

Comment:
Since Uncoiled Twinfang/Twinblood are directly tied to Uncoiled Fury, I feel like this text change makes the checklist header overly wordy. If they player is dropping the followup oGCDs, they'll see that context in the expanded entry/the requirements. I think it'd make sense to leave the checklist name as it was.

Reply:
reverted checklist name